app.py

```diff
@@ -574,6 +574,7 @@ def the_blast_promo_form():
         form=form,
         campaign_id=campaign_id,
         referral_id=referral_id,
+        amount="250",
```

Comment:
**praise:** Great solve for this! I like that we can control the promotional amount in one line and have it cascade to the form and display text. Smart!

Reply:
Woo, thanks and glad to hear it.